src/main/java/com/avioconsulting/mule/logger/internal/CustomLogger.java

```diff
@@ -7,10 +7,9 @@
 import com.avioconsulting.mule.logger.api.processor.MessageAttributes;
 import com.avioconsulting.mule.logger.internal.config.CustomLoggerConfiguration;
 import com.avioconsulting.mule.logger.internal.utils.CustomLoggerUtils;
+import java.text.SimpleDateFormat;
 import java.time.Instant;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
In general we should avoid wildcard imports and keep specific classes

Reply:
I think intellij automatically did this bc we had a growing number of java.util imports. But fixed this as well